app/controllers/admin/tutorials_controller.rb

```diff
@@ -22,11 +35,11 @@ def destroy
     Tutorial.destroy(params[:id])
     flash[:success] = "Tutorial Deleted."
 
-    redirect_to admin_dashboard_path
+    redirect_to '/admin/dashboard'
```

Comment:
Why did we use the actual route instead of the path helper here?

Reply:
it wasn't working with the path helper so i hand rolled it. We will have to chase this bug later anyway